### PR TITLE
docs: show border values as objects

### LINKED ISSUE #4002: Documentation for theme objects should clarify the object structure for certain properties
## Current Behavior

Documentation for border200 (or other border properties) and radius200 doesn't differentiate whether the properties are objects or primitives.

```
theme.borders.border200
solid
1px
hsla(0, 0%, 0%, 0.08)
```

```
theme.borders.radius200
4px
```

## Expected Behavior

The documentation should explain the structure of the properties better.

```
theme.borders.border200
{ 
  borderColor: "hsla(0, 0%, 0%, 0.08)"
  borderStyle: "solid"
  borderWidth: "1px"
}
```

```
theme.borders.radius200
4px
```

## Your Environment

https://baseweb.design/guides/theming/#borders

- [x] I have searched the [issues](https://github.com/uber/baseweb/issues) of this repository and believe that this is not a duplicate.

### PR DESCRIPTION
**Pull request by tajo**
fixes #4002 

Screen Shot 2021-01-04 at 6 28 30 PM

### CODE REVIEW

**Review by chasestarr — APPROVED** [review]
Is this the only theme value that is an object?

**Comment by tajo** [comment]
> Is this the only theme value that is an object?

Also updated `font` and `grid` props.

File: documentation-site/components/theming/borders.js [hunk 1]
```diff
@@ -25,11 +25,19 @@ export function Border({name}: {name: string}) {
         ></div>
       )}
       renderValue={({previewTheme}) => (
-        <React.Fragment>
-          <div>{previewTheme.borders[name].borderStyle}</div>
-          <div>{previewTheme.borders[name].borderWidth}</div>
-          <div>{previewTheme.borders[name].borderColor}</div>
-        </React.Fragment>
```
Comment (ajbogh) [inline documentation-site/components/theming/borders.js:32]: Why not use 

```
<pre>
{JSON.stringify(previewTheme.borders[name], null, 2)}
</pre>
```

File: documentation-site/components/theming/typography.js [hunk 2]
```diff
@@ -20,11 +20,19 @@ export function Type({name}: {name: string}) {
         <div className={css({...theme.typography[name]})}>Example</div>
       )}
       renderValue={() => (
-        <React.Fragment>
-          <div>{theme.typography[name].fontSize}</div>
-          <div>{theme.typography[name].fontWeight}</div>
-          <div>{theme.typography[name].lineHeight}</div>
-        </React.Fragment>
```
Comment (ajbogh) [inline documentation-site/components/theming/typography.js:27]: Might be easier to use:

```
<pre>
{JSON.stringify(theme.typography[name], null, 2)}
</pre>
```

**Pull request merged**